packages/design-system/src/components/combobox.tsx

```diff
@@ -134,6 +153,7 @@ type ComboboxProps<Item> = {
 export const Combobox = <Item extends BaseItem>({
   items,
   value,
+  selected = value,
```

Comment:
so you want value and selected value to be 2 different things? how does this fit with the API redesign @okonet ?

Reply:
It doesn't. Why is this needed?

Reply:
so representation of the value inside the input.value is different than the value inside the items. It seems like a case for the itemToString function that would return the value without the unit, so it can be the same

Reply:
Generally we should avoid specific use cases to introduce breaking changes in reusable components. 

I'm still not sure what problem you're trying to solve by this so maybe a small example in a Storybook so I could help you?

Reply:
I believe this is a misused component case here. Unit should be a Select with a customized trigger component (IconButton) inside a `suffix` of the TextField. This is pretty much how it works now.

Reply:
So it would be a <Combobox suffix={<Select />} />, where combobox always receives values without units and Select always receives units without values